.github/workflows/scancode.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# reusable workflow to call scancode action to scan licenses and copyrights and
+# upload the results.
+# TODO: add more usage guide here.
+name: 'scancode'
```

Comment:
We will need a caller workflow in the project repo, and the result will look like below, where you can download the json and csv file in your laptop:
<img width="955" alt="Screenshot 2023-10-27 at 10 31 00 AM" src="https://github.com/abcxyz/pkg/assets/51539171/7f0d8f70-c9b3-42e5-8a5a-667885b17cbb">